registry.json

```diff
@@ -474,7 +474,7 @@
     "id": "ethereum",
     "name": "Ethereum",
     "coinId": 60,
-    "symbol": "ETH",
+    "symbol": "BTC",
```

Comment:
hey @satoshiotomakan, I suppose it should be done for Merlin chain (this is an object for Ethereum)